### PR TITLE
Add two legal pages for anbox cloud and add them to the terms and policies list page

### PR DESCRIPTION
**Pull request by deadlight**
## Done

- Added terms page
- Added privacy notice page
- Updated terms list

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: 
  - http://0.0.0.0:8001/legal/terms-and-policies
  - http://0.0.0.0:8001/legal/anbox-cloud-terms-of-service
  - http://0.0.0.0:8001/legal/anbox-cloud-privacy-notice
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/6605

### CODE REVIEW

File: templates/legal/anbox-cloud-terms-of-service.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      </section>
+      <section>
+        <h2 id="general">9. General</h2>
+        <p>These Terms of Service are governed by the laws of England and any dispute will be heard by the courts in England.</p>
+        <p>Failure by Canonical to enforce any right or provision of these Terms of Service shall not constitute a waiver of such right or provision. If any part of these Terms of Service is held invalid or unenforceable, that part will be construed to reflect the parties original intent, and the remaining portions will remain in full force and effect. The terms of these Terms of Service do not affect your statutory rights.</p>
+        <p>Any notices should be sent by email to <a href="mailto:legal@canonical.com">legal@canonical.com</a> or by registered post to:</p>
+        <p>Canonical Group Limited,<br>
+        5th Floor, Blue Fin Building,<br>
+        110 Southwark Street,<br>
+        London, SE1 0SU.</p>
```
Comment (sowasred2012) [inline templates/legal/anbox-cloud-terms-of-service.html]: I know it's in the copy doc but the pedant in me is pretty sure the period isn't needed in this instance:

```suggestion
        London, SE1 0SU</p>
```

File: templates/legal/anbox-cloud-privacy-notice.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+          <li class="p-list__item is-ticked">require the erasure of personal information concerning you in certain situations;</li>
+          <li class="p-list__item is-ticked">receive the personal information concerning you which you have provided to us, in a structured, commonly used and machine-readable format and have the right to transmit that data to a third party in certain situations;</li>
+          <li class="p-list__item is-ticked">object at any time to processing of personal information concerning you for direct marketing;</li>
+          <li class="p-list__item is-ticked">object to decisions being taken by automated means which produce legal effects concerning you or similarly significantly affect you;</li>
+          <li class="p-list__item is-ticked">object in certain other situations to our continued processing of your personal information;</li>
+          <li class="p-list__item is-ticked">otherwise restrict our processing of your personal information in certain circumstances;</li>
+          <li class="p-list__item is-ticked">claim compensation for damages caused by out breach of any data protection laws.</li>
```
Comment (sowasred2012) [inline templates/legal/anbox-cloud-privacy-notice.html]:
```suggestion
          <li class="p-list__item is-ticked">claim compensation for damages caused by our breach of any data protection laws.</li>
```

**Comment by anthonydillon** [comment]
@deadlight can you apply Scotts suggestions then I will review again.

**Comment by deadlight** [comment]
@anthonydillon ready

File: templates/legal/anbox-cloud-privacy-notice.html [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+      <section>
+        <h2 id="your-right-to-complain">Your right to complain</h2>
+        <p>We hope that we can resolve any query or concern you raise about our use of your information. You can contact us at any time. Any questions, comments and requests regarding this Privacy notice or any other related matter are welcomed and should be addressed to <a href="mailto:dataprotection@canonical.com">dataprotection@canonical.com</a> or to the address below:</p>
+        <p>Legal, Canonical<br>
+        5th Floor, Blue Fin Building<br>
+        110 Southwark Street<br>
+        London, SE1 0SU<br>
+        United Kingdom</p>
+        <p>Alternatively, you can use the relevant contact us form.</p>
+        <p>If you have a complaint about our use of your information, you can contact the Information Commissioner’s Office via their website at <a href="ico.org.uk/make-a-complaint">ico.org.uk/make-a-complaint</a> or write to them at:</p>
```
Comment (anthonydillon) [inline templates/legal/anbox-cloud-privacy-notice.html]: This link is going to need `https://` in front of it.

**Comment by anasereijo** [comment]
Looks good, thanks!

**Pull request merged**